osi_detectedobject.proto

```diff
@@ -161,4 +223,99 @@ message DetectedObject
         //
         optional double prob_animal = 9;
     }
+
+    // Definition of measurement states.
+    //
+    enum MeasurementState
```

Comment:
Would it make sense to move this enum also to the DetectedObjectHeader, which would make reuse simpler (see my comments in PR #111 )?